app/ui/browser/model/index.js

```diff
@@ -14,6 +14,11 @@ import Immutable from 'immutable';
 import uuid from 'uuid';
 
 /**
+ * Fairly sure we should hard code this
```

Comment:
I'm actively pursuing having the main process inject the set of pages into each created window process for session restore.  I don't think that will impact your approach.